doc/onboarding/model/form.md

```diff
@@ -0,0 +1,30 @@
+# `form` State
+
+## Definition
+State of a form that includes its values but also :
+ - required field or
+ - ability to submit or not all the form.
+
+```
+{
+    <boolean> disabled: false
```

Comment:
I asked to @jsilvestre and @poupotte : it seems that this props is useless which means that form could be submitted be server would send an error if passwordValue will be too weak.